unpackaged/vesuvio_calibration/tests/system/test_system_analysis.py

```diff
@@ -246,10 +264,12 @@ def _assert_E1_parameters_match_expected(self, params_table, rel_tolerance, fit_
             global_E1 = params_table.column('global_E1')[0]
             self.assertAlmostEqual(global_E1, EVSGlobals.ENERGY_ESTIMATE, delta=EVSGlobals.ENERGY_ESTIMATE*rel_tolerance)
 
-    def _assert_parameters_match_expected(self, params_table, tolerances=None):
+    def _assert_parameters_match_expected(self, params_table, tolerances=None, fit_type="Individual"):
         rel_tol_theta, rel_tol_L1 = self._extract_tolerances(deepcopy(tolerances))
         theta_errors = self._assert_theta_parameters_match_expected(params_table, rel_tol_theta)
-        L1_errors = self._assert_L1_parameters_match_expected(params_table, rel_tol_L1)
+        L1_errors = None
+        if fit_type != 'Shared':
```

Comment:
Remind me, why can't we test L1 parameters if the fit is shared?

Reply:
I've tried writing tests for the global L1 values (which are calculated using the global E1 values), but found that the values didn't match the expected L1 parameters - I was going to bring this up with Matthew when we next speak to him to check that the global L1 values are being calculated in the correct way

Reply:
Ok sounds good, I'll take another look at this PR when I get a chance, we're currently in our release sprint so I have a bit of a to-do list!